data/tgst5.dat

```diff
@@ -25287,7 +25287,7 @@
 {"dn": 129, "lib": "ut;Nostra13 Image Loading;https://github.com/nostra13/Android-Universal-Image-Loader", "sp": "com/nostra13/universalimageloader/utils", "bh": 43314, "btc": 24, "btn": 17, "pn": "com/nostra13"}
 {"dn": 106, "lib": "", "sp": "com/myapphone/android/modules/ticket", "bh": 33442, "btc": 24, "btn": 17}
 {"dn": 259, "lib": "ge;Andengine Game Engine;http://www.andengine.org/", "sp": "org/anddev/andengine/extension/input/touch", "bh": 32251, "btc": 24, "btn": 17, "pn": "org/anddev/andengine"}
-{"dn": 125, "lib": "", "sp": "com/adjust/sdk/plugin", "bh": 29808, "btc": 24, "btn": 17}
+{"dn": 125, "lib": "ma;Adjust SDK;https://github.com/adjust/android_sdk", "sp": "com/adjust/sdk/plugin", "bh": 29808, "btc": 24, "btn": 17, "pn": "com/adjust/sdk"}
```

Comment:
Already tagged in V2 🤗
Lcom/adjust/sdk,Adjust SDK,Mobile Analytics,https://www.adjust.com/